cmd/boulder-wfe/main.go

```diff
@@ -108,9 +108,9 @@ func main() {
 	wfe.IssuerCacheDuration, err = time.ParseDuration(c.WFE.IssuerCacheDuration)
 	cmd.FailOnError(err, "Couldn't parse issuer caching duration")
 
-	wfe.ShutdownStopTimeout, err = time.ParseDuration(c.WFE.ShutdownStopTimeout)
+	shutdownStopTimeout, err := time.ParseDuration(c.WFE.ShutdownStopTimeout)
```

Comment:
nit: slightly out of the scope of this change but we already have a fancy `cmd.ConfigDuration` which will do the parsing etc for us, would you mind switching these strings out for that in `cmd/boulder-wfe/main.go`.